qdrant-landing/content/documentation/faq/qdrant-fundamentals.md

```diff
@@ -66,6 +66,10 @@ But in some cases, we might be able to help you with that through manual interve
 
 ## Versioning
 
+### Do you support downgrade?
+
+We does not support cluster downgrade. This applies to both cloud and core product.
```

Comment:
Maybe it would be nice to include the reasoning?

```suggestion
We does not support cluster downgrade. This applies to both cloud and core products. Deploying a newer version of Qdrant will automatically migrate your data to the newer storage format, which is not reversible.
```

Reply:
Question: what happens if someone has their own backup and installs a new instance of an old version of Qdrant? Example, with a config file such as:
```
services:
  qdrant:
    image: qdrant/qdrant: 1.7
```

Reply:
The backup was made on 1.7 or 1.8?

Reply:
In this case, presumably 1.7

Reply:
I would suppose that it's would work. But i don't know if we should promote this , @mac-chaffee any thoughts about this?

Reply:
Yeah if you take a physical backup from v1.7 and then restore it to a qdrant v1.7 cluster, then yes it would work...

If you take a physical backup from v1.7 and then restore it to a qdrant v1.7 cluster, this would also work, since Qdrant would upgrade the data format at start time.

If you take a physical backup from v1.8 and then restore it to a qdrant v1.7 cluster, this would not work, since Qdrant can't downgrade its own storage format.

Reply:
> But i don't know if we should promote this

Are we still supporting v1.7?

Reply:
>  But i don't know if we should promote this 

Thinking this through, maybe not. Users who understand downgrades, especially of databases, should already understand the scenarios that @mac-chaffee described. Approving.